doc_test.go

```diff
@@ -289,6 +289,19 @@ func Example_rasterTutorial() {
 	poDstDS.Bands()[0].Write(0, 0, abyRaster, 512, 512)
 	poDstDS.Close()
 
+	poDstDSInt8, err := godal.Create(godal.GTiff, pszDstFilename, 1, godal.Int8, 512, 512)
```

Comment:
Please put this test inside a new function in godal_test.go. doc_test.go is meant for documenting basic usage